README.md

```diff
@@ -34,10 +34,18 @@ Contact Form Plugin
 6. Define parameters in `.env` file
 
     ```
+   # Define if an email should be send to the manager when contact form is send
+    CONTACT_FORM_SEND_MANAGER_MAIL=
```

Comment:
Only variables that need to be changed in different environments are put here.
Only: `CONTACT_FORM_RECAPTCHA_PUBLIC_KEY` and `CONTACT_FORM_RECAPTCHA_SECRET_KEY`

Leave the others in the parameters

Reply:
DONE